rust/libnewsboat/src/utils.rs

```diff
@@ -1037,6 +1037,33 @@ pub fn locale_to_utf8(text: &[u8]) -> String {
     String::from_utf8(converted).expect("convert_text() returned a non-UTF-8 string")
 }
 
+/// Parses a string in the form `author@example.com (Example Author)`
+/// into a pair of (author, email).
+/// If the pattern cannot be matched or the name is empty, the input
+/// is returned completely as author with empty email.
+///
+/// Invalid data (outside of utf-8) is dropped
+pub fn parse_rss_author_email(text: &[u8]) -> (String, String) {
+    let re = regex::bytes::Regex::new(r#"(.+@.+) \((.+)\)"#)
```

Comment:
I have not checked what the standard says, but should we allow for multiple spaces with `\s+` (or whatever this particular regex engine uses) to separate the e-mail address and name in parentheses?

Reply:
Good idea. Apparently that was working already (accidentally) because the second `.+` in the email was matching the extra whitespace and the `trim()` would remove it from the email.

However, I think it is better to be explicit about it so it is updated now